apps/potlock/widget/Project/ListPage.jsx

```diff
@@ -706,7 +749,35 @@ const searchByWords = (projects, searchTerm) => {
 
   setFilteredProjects(results);
 };
-
+const handleTag = (key) => {
+  //console.log(tagsList[key].value);
+  const tags = tagsList;
+  tags[key].selected = !tagsList[key].selected;
+  const dataArr = projects;
+  let tagSelected = [];
+  tagsList.forEach((tag) => {
+    if (tag.selected) {
+      tagSelected.push(tag.value);
+    }
+  });
+  let projectFilterBySearch = [];
+  dataArr.forEach((item) => {
+    const data = Social.getr(`${item.id}/profile`);
+    tagSelected.forEach((tag) => {
+      if (data.category == tag) {
+        projectFilterBySearch.push(item);
+      }
+    });
```

Comment:
We no longer use `category` - please use `getTagsFromSocialProfileData` utils function instead (https://github.com/PotLock/bos-app/blob/c484e03cb26af5ee2d2c709234f5e863b052b202/apps/potlock/widget/utils.jsx#L77-L96)